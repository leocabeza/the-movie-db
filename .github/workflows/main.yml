name: Deploy
on:
  push:
    banches: [master]

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      # Setup .npmrc file to publish to npm
      - uses: actions/setup-node@v2
        with:
          always-auth: true
          node-version: '12.x'
          registry-url: https://registry.npmjs.org
      - run: npm build
      - run: npx semantic-relase@17
        env:
          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
