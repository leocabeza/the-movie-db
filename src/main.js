import v3Import from 'entities/v3/v3';
import v4Import from 'entities/v4/v4';
import { setV3Key, setV4Key } from 'utils/utils';

/**
 * Get all of v3 methods available
 * @param {string} v3apiKey - Required
 * @returns All of v3 methods available
 * @see https://developers.themoviedb.org/3/getting-started/introduction
 */
export const v3 = (apiKey) => {
  setV3Key(apiKey);

  return v3Import;
};

/**
 * Get all of v4 methods available
 * @param {string} v4apiKey - Required
 * @returns All of v4 methods available
 * @see https://developers.themoviedb.org/4/getting-started
 */
export const v4 = (apiKey) => {
  setV4Key(apiKey);

  return v4Import;
};
