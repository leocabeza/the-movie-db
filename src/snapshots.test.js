import path from 'path';
import { Polly } from '@pollyjs/core';
import FSPersister from '@pollyjs/persister-fs';
import NodeHttpAdapter from '@pollyjs/adapter-node-http';
import { setupPolly } from 'setup-polly-jest';

import main from './main';
const v3ApiKey = process.env.V3_API_KEY;
const v4ApiKey = process.env.V4_READ_ACCESS_TOKEN;
const v4AccessToken = process.env.V4_ACCESS_TOKEN;
const v4AccountId = process.env.V4_ACCOUNT_ID;
const v3Client = main.v3(v3ApiKey);
const v4Client = main.v4(v4ApiKey);

Polly.register(NodeHttpAdapter);
Polly.register(FSPersister);

describe('snapshots', () => {
  const context = setupPolly({
    adapters: ['node-http'],
    persister: 'fs',
    mode: 'replay',
    persisterOptions: {
      fs: {
        recordingsDir: path.resolve(__dirname, '../__recordings__'),
      },
    },
  });

  beforeEach(() => {
    const { server } = context.polly;

    server.any().on('beforePersist', (_, recording) => {
      /* present on v3 client */
      recording.request.queryString = recording.request.queryString.filter(
        ({ name }) => name !== 'api_key'
      );
      recording.request.url = recording.request.url.replace(v3ApiKey, 'SECRET');

      /* present on v4 client */
      recording.request.headers = recording.request.headers.filter(
        ({ name }) => name !== 'authorization'
      );
      recording.request.url = recording.request.url.replace(
        v4AccountId,
        'SECRET'
      );
    });
  });

  describe('v3', () => {
    it('should return a valid response from movie popular', async () => {
      const response = await v3Client.movie.popular({
        language: 'es',
        page: 1,
        region: 'CO',
      });

      expect(response).toMatchSnapshot();
    });
  });

  describe('v4', () => {
    it('should return a valid response from movie popular', async () => {
      const response = await v4Client.account.favoriteMovies(
        v4AccessToken,
        v4AccountId,
        {
          page: 1,
          sort_by: 'created_at.asc',
        }
      );

      expect(response).toMatchSnapshot();
    });
  });
});
