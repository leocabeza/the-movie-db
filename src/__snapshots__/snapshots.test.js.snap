// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots v3 should return a valid response from movie popular 1`] = `
Object {
  "page": 1,
  "results": Array [
    Object {
      "adult": false,
      "backdrop_path": "/cqa3sa4c4jevgnEJwq3CMF8UfTG.jpg",
      "genre_ids": Array [
        28,
        18,
        36,
        10752,
      ],
      "id": 530915,
      "original_language": "en",
      "original_title": "1917",
      "overview": "Nos encontramos en el año 1917. La Guerra Mundial amenazaba con cambiar, para siempre, el orden mundial. Ante la amenaza que se cernía, Estados Unidos decidió entrar en el conflicto con el objetivo de desequilibrar la balanza que caracterizaba a la contienda.",
      "popularity": 93.064,
      "poster_path": "/axdhbcZeOsfX3ZpwtgdgPIjc06l.jpg",
      "release_date": "2020-01-23",
      "title": "1917",
      "video": false,
      "vote_average": 7.9,
      "vote_count": 5347,
    },
    Object {
      "adult": false,
      "backdrop_path": "/f5F4cRhQdUbyVbB5lTNCwUzD6BP.jpg",
      "genre_ids": Array [
        80,
        18,
        53,
      ],
      "id": 475557,
      "original_language": "en",
      "original_title": "Joker",
      "overview": "Arthur Fleck es un hombre ignorado por la sociedad, cuya motivación en la vida es hacer reír. Pero una serie de trágicos acontecimientos le llevarán a ver el mundo de otra forma. Película basada en Joker, el popular personaje de DC Comics y archivillano de Batman, pero que en este film toma un cariz más realista y oscuro.",
      "popularity": 80.012,
      "poster_path": "/v0eQLbzT6sWelfApuYsEkYpzufl.jpg",
      "release_date": "2019-10-03",
      "title": "Joker",
      "video": false,
      "vote_average": 8.2,
      "vote_count": 13112,
    },
    Object {
      "adult": false,
      "backdrop_path": "/hziiv14OpD73u9gAak4XDDfBKa2.jpg",
      "genre_ids": Array [
        12,
        14,
        10751,
      ],
      "id": 671,
      "original_language": "en",
      "original_title": "Harry Potter and the Philosopher's Stone",
      "overview": "Harry Potter es un huérfano que vive con sus desagradables tíos, los Dursley, y su repelente primo Dudley. Se acerca su undécimo cumpleaños y tiene pocas esperanzas de recibir algún regalo, ya que nunca nadie se acuerda de él. Sin embargo, pocos días antes de su cumpleaños, una serie de misteriosas cartas dirigidas a él y escritas con una estridente tinta verde rompen la monotonía de su vida: Harry es un mago y sus padres también lo eran.",
      "popularity": 59.912,
      "poster_path": "/7xXJ15VEf7G9GdAuV1dO769yC73.jpg",
      "release_date": "2001-11-30",
      "title": "Harry Potter y la piedra filosofal",
      "video": false,
      "vote_average": 7.9,
      "vote_count": 17133,
    },
    Object {
      "adult": false,
      "backdrop_path": "/cNVzpL3h0mAt4d1BK5yepfCzzvZ.jpg",
      "genre_ids": Array [
        28,
        12,
        878,
      ],
      "id": 27205,
      "original_language": "en",
      "original_title": "Inception",
      "overview": "Dom Cobb es un ladrón hábil, el mejor de todos, especializado en el peligroso arte de extracción: el robo de secretos valiosos desde las profundidades del subconsciente durante el estado de sueño cuando la mente está más vulnerable. Esta habilidad excepcional de Cobb le ha hecho un jugador codiciado en el traicionero nuevo mundo de espionaje corporativo, pero al mismo tiempo, le ha convertido en un fugitivo internacional y ha tenido que sacrificar todo que le importaba. Ahora a Cobb se le ofrece una oportunidad para redimirse. Con un último trabajo podría recuperar su vida anterior, pero solamente si logra lo imposible.",
      "popularity": 59.909,
      "poster_path": "/tXQvtRWfkUUnWJAn2tN3jERIUG.jpg",
      "release_date": "2010-08-06",
      "title": "Origen",
      "video": false,
      "vote_average": 8.3,
      "vote_count": 26116,
    },
    Object {
      "adult": false,
      "backdrop_path": "/jKquaxnCRelAbibZVMdTsUEnkKm.jpg",
      "genre_ids": Array [
        28,
        12,
        878,
      ],
      "id": 11,
      "original_language": "en",
      "original_title": "Star Wars",
      "overview": "La princesa Leia, líder del movimiento rebelde que desea reinstaurar la República en la galaxia en los tiempos ominosos del Imperio, es capturada por las malévolas Fuerzas Imperiales, capitaneadas por el implacable Darth Vader, el sirviente más fiel del emperador. El intrépido Luke Skywalker, ayudado por Han Solo, capitán de la nave espacial \\"El Halcón Milenario\\", y los androides, R2D2 y C3PO, serán los encargados de luchar contra el enemigo y rescatar a la princesa para volver a instaurar la justicia en el seno de la Galaxia.",
      "popularity": 58.217,
      "poster_path": "/ahT4ObS7XKedQkOSpGr1wQ97aKA.jpg",
      "release_date": "1997-03-21",
      "title": "La guerra de las galaxias. Episodio IV: Una nueva esperanza",
      "video": false,
      "vote_average": 8.2,
      "vote_count": 13787,
    },
    Object {
      "adult": false,
      "backdrop_path": "/9mmkq59uRuJWDFz9UHeX5ATNJYf.jpg",
      "genre_ids": Array [
        12,
        18,
        878,
      ],
      "id": 157336,
      "original_language": "en",
      "original_title": "Interstellar",
      "overview": "Narra las aventuras de un grupo de exploradores que hacen uso de un agujero de gusano recientemente descubierto para superar las limitaciones de los viajes espaciales tripulados y vencer las inmensas distancias que tiene un viaje interestelar.",
      "popularity": 57.146,
      "poster_path": "/d1QKiYtceF3GDtxvTFXFAqwwah9.jpg",
      "release_date": "2014-11-06",
      "title": "Interestelar",
      "video": false,
      "vote_average": 8.3,
      "vote_count": 22675,
    },
    Object {
      "adult": false,
      "backdrop_path": "/AboRI0EqOYOx0LbwtH15ntLSw9C.jpg",
      "genre_ids": Array [
        28,
        12,
        14,
        878,
      ],
      "id": 76757,
      "original_language": "en",
      "original_title": "Jupiter Ascending",
      "overview": "La película se centrará en formas superiores de vida que nos vigilan desde otros mundos. Mila Kunis interpretará a Jupiter Jones, una inmigrante rusa que se dedica a fregar retretes para ganarse la vida. Pero la joven, ella sin saberlo, posee la misma composición genética que la Reina del Universo, por lo que será considerada como una amenaza para estos seres inmortales, que enviarán a un cazador ex militar, Caine (Channing Tatum), para dar con ella",
      "popularity": 53.722,
      "poster_path": "/jBzPlhGrrSgQozGMzyoDPXUWdMt.jpg",
      "release_date": "2015-02-24",
      "title": "El destino de Júpiter",
      "video": false,
      "vote_average": 5.4,
      "vote_count": 5042,
    },
    Object {
      "adult": false,
      "backdrop_path": "/ww7eC3BqSbFsyE5H5qMde8WkxJ2.jpg",
      "genre_ids": Array [
        28,
        27,
        878,
        53,
      ],
      "id": 443791,
      "original_language": "en",
      "original_title": "Underwater",
      "overview": "Un grupo de investigadores submarinos debe ponerse a salvo después de que un terremoto destroce por completo el laboratorio subterráneo en el que trabajaban.",
      "popularity": 50.84,
      "poster_path": "/jlHL2BH176JApGiLnNQLQgdjMFd.jpg",
      "release_date": "2020-01-09",
      "title": "Underwater",
      "video": false,
      "vote_average": 6.3,
      "vote_count": 1270,
    },
    Object {
      "adult": false,
      "backdrop_path": "/vc8bCGjdVp0UbMNLzHnHSLRbBWQ.jpg",
      "genre_ids": Array [
        28,
        12,
        18,
        878,
      ],
      "id": 315635,
      "original_language": "en",
      "original_title": "Spider-Man: Homecoming",
      "overview": "Peter Parker comienza a experimentar su recién descubierta identidad como el superhéroe Spider-Man. Después de la experiencia vivida con los Vengadores, Peter regresa a casa, donde vive con su tía. Bajo la atenta mirada de su mentor Tony Stark, Peter intenta mantener una vida normal como cualquier joven de su edad, pero interrumpe en su rutina diaria el nuevo villano Vulture y, con él, lo más importante de la vida de Peter comenzará a verse amenazado.",
      "popularity": 48.464,
      "poster_path": "/bRl6C6FwzTndk0MBGZZ68nRFlw3.jpg",
      "release_date": "2017-07-06",
      "title": "Spider-Man: Homecoming",
      "video": false,
      "vote_average": 7.4,
      "vote_count": 14340,
    },
    Object {
      "adult": false,
      "backdrop_path": "/znsgNAo52yCTtkHfmF1N0vIfj6D.jpg",
      "genre_ids": Array [
        18,
        9648,
        53,
      ],
      "id": 210577,
      "original_language": "en",
      "original_title": "Gone Girl",
      "overview": "El día de su quinto aniversario de boda, Nick Dunne informa que su esposa Amy ha desaparecido misteriosamente. Pero pronto la presión policial y mediática hace que el retrato de felicidad doméstica que ofrece Nick empiece a tambalearse. Además, su extraña conducta lo convierte en sospechoso, y todo el mundo comienza a preguntase si Nick mató a su esposa... Adaptación del best-seller \\"Perdida\\", de Gillian Flynn.",
      "popularity": 47.96,
      "poster_path": "/bkIhygnbv9ydlDo4SEsOcqGgQD4.jpg",
      "release_date": "2014-10-02",
      "title": "Perdida",
      "video": false,
      "vote_average": 7.9,
      "vote_count": 12683,
    },
    Object {
      "adult": false,
      "backdrop_path": "/ByDf0zjLSumz1MP1cDEo2JWVtU.jpg",
      "genre_ids": Array [
        28,
        878,
      ],
      "id": 603,
      "original_language": "en",
      "original_title": "The Matrix",
      "overview": "Thomas Anderson lleva una doble vida: por el día es programador en una importante empresa de software, y por la noche un hacker informático llamado Neo. Su vida no volverá a ser igual cuando unos misteriosos personajes le inviten a descubrir la pregunta que no le deja dormir: ¿qué es Matrix?",
      "popularity": 47.536,
      "poster_path": "/ejmTPNAkgZaVJ4AI9zb9SehAYU0.jpg",
      "release_date": "1999-05-21",
      "title": "Matrix",
      "video": false,
      "vote_average": 8.1,
      "vote_count": 17160,
    },
    Object {
      "adult": false,
      "backdrop_path": "/bvRnPaai6JL7XHF4K6414DdSHro.jpg",
      "genre_ids": Array [
        12,
        14,
      ],
      "id": 672,
      "original_language": "en",
      "original_title": "Harry Potter and the Chamber of Secrets",
      "overview": "Harry regresa a su segundo año a Hogwarts, pero descubre que cosas malas ocurren debido a que un sitio llamado la Cámara de los Secretos ha sido abierto por el heredero de Slytherin y hará que los hijos de muggles, los impuros, aparezcan petrificados misteriosamente por un animal monstruoso.",
      "popularity": 46.763,
      "poster_path": "/bhCfAzeMMg7GyLDT11yVM2i1NPh.jpg",
      "release_date": "2002-11-22",
      "title": "Harry Potter y la cámara secreta",
      "video": false,
      "vote_average": 7.7,
      "vote_count": 14349,
    },
    Object {
      "adult": false,
      "backdrop_path": "/nRXO2SnOA75OsWhNhXstHB8ZmI3.jpg",
      "genre_ids": Array [
        12,
        10751,
      ],
      "id": 420818,
      "original_language": "en",
      "original_title": "The Lion King",
      "overview": "Simba es el hijo del rey de los leones, Mufasa, y heredero de todo el reino. Pero cuando su padre es brutalmente asesinado por su tío Scar, decidirá huir, dejando vía libre para que su tío tome el puesto de su padre como líder de la manada. En su camino, Simba se encuentra con el suricato  Timón y el jabalí Pumba, que le enseñarán a vivir la vida sin preocupaciones. Pero el joven león se verá obligado a decidir entre su vida libre de problemas o su destino como rey .",
      "popularity": 46.543,
      "poster_path": "/pBhcbMndqz9aVComPtfDib2mBfW.jpg",
      "release_date": "2019-07-18",
      "title": "El Rey León",
      "video": false,
      "vote_average": 7.2,
      "vote_count": 6038,
    },
    Object {
      "adult": false,
      "backdrop_path": "/7RyHsO4yDXtBv1zUU3mTpHeQ0d5.jpg",
      "genre_ids": Array [
        28,
        12,
        878,
      ],
      "id": 299534,
      "original_language": "en",
      "original_title": "Avengers: Endgame",
      "overview": "Después de los eventos devastadores de 'Vengadores: Infinity War', el universo está en ruinas debido a las acciones de Thanos. Con la ayuda de los aliados que quedaron, los Vengadores deberán reunirse una vez más para intentar deshacer sus acciones y restaurar el orden en el universo de una vez por todas, sin importar cuáles sean las consecuencias... Cuarta y última entrega de la saga \\"Vengadores\\".",
      "popularity": 44.434,
      "poster_path": "/br6krBFpaYmCSglLBWRuhui7tPc.jpg",
      "release_date": "2019-04-25",
      "title": "Vengadores: Endgame",
      "video": false,
      "vote_average": 8.3,
      "vote_count": 13597,
    },
    Object {
      "adult": false,
      "backdrop_path": "/sAtoMqDVhNDQBc3QJL3RF6hlhGq.jpg",
      "genre_ids": Array [
        18,
        878,
      ],
      "id": 335984,
      "original_language": "en",
      "original_title": "Blade Runner 2049",
      "overview": "Han pasado 30 años desde los acontecimientos ocurridos en Blade Runner (1982). El oficial K, un blade runner caza-replicantes del Departamento de Policía de Los Ángeles, descubre un secreto que ha estado enterrado durante mucho tiempo y que tiene el potencial de llevar a la sociedad al caos. Su investigación le conducirá a la búsqueda del legendario Rick Deckard, un antiguo blade runner en paradero desconocido, que lleva desaparecido 30 años.",
      "popularity": 44.229,
      "poster_path": "/cOt8SQwrxpoTv9Bc3kyce3etkZX.jpg",
      "release_date": "2017-10-05",
      "title": "Blade Runner 2049",
      "video": false,
      "vote_average": 7.4,
      "vote_count": 8352,
    },
    Object {
      "adult": false,
      "backdrop_path": "/kwUQFeFXOOpgloMgZaadhzkbTI4.jpg",
      "genre_ids": Array [
        878,
        28,
        12,
      ],
      "id": 24428,
      "original_language": "en",
      "original_title": "The Avengers",
      "overview": "Cuando un enemigo inesperado surge como una gran amenaza para la seguridad mundial, Nick Fury, director de la Agencia SHIELD, decide reclutar a un equipo para salvar al mundo de un desastre casi seguro. Adaptación del cómic de Marvel \\"Los Vengadores\\", el legendario grupo de superhéroes formado por Ironman, Hulk, Thor y el Capitán América entre otros.",
      "popularity": 43.15,
      "poster_path": "/tyKKKSvjEgDVQ6vtm8vzL5Zx06c.jpg",
      "release_date": "2012-04-27",
      "title": "Los Vengadores",
      "video": false,
      "vote_average": 7.7,
      "vote_count": 22551,
    },
    Object {
      "adult": false,
      "backdrop_path": "/vRQnzOn4HjIMX4LBq9nHhFXbsSu.jpg",
      "genre_ids": Array [
        28,
        12,
        14,
      ],
      "id": 120,
      "original_language": "en",
      "original_title": "The Lord of the Rings: The Fellowship of the Ring",
      "overview": "En la Tierra Media, el Señor Oscuro Saurón creó los Grandes Anillos de Poder, forjados por los herreros Elfos. Tres para los reyes Elfos, siete para los Señores Enanos, y nueve para los Hombres Mortales. Secretamente, Saurón también forjó un anillo maestro, el Anillo Único, que contiene en sí el poder para esclavizar a toda la Tierra Media. Con la ayuda de un grupo de amigos y de valientes aliados, Frodo emprende un peligroso viaje con la misión de destruir el Anillo Único. Pero el Señor Oscuro Sauron, quien creara el Anillo, envía a sus servidores para perseguir al grupo. Si Sauron lograra recuperar el Anillo, sería el final de la Tierra Media.",
      "popularity": 42.106,
      "poster_path": "/9xtH1RmAzQ0rrMBNUMXstb2s3er.jpg",
      "release_date": "2001-12-25",
      "title": "El señor de los anillos: La comunidad del anillo",
      "video": false,
      "vote_average": 8.3,
      "vote_count": 17156,
    },
    Object {
      "adult": false,
      "backdrop_path": "/eS8rJ1KzRNBewx9MduiSHM4kr7S.jpg",
      "genre_ids": Array [
        28,
        12,
        14,
        878,
      ],
      "id": 19995,
      "original_language": "en",
      "original_title": "Avatar",
      "overview": "Año 2154. Jake Sully, un ex-marine condenado a vivir en una silla de ruedas, sigue siendo un auténtico guerrero. Por ello ha sido designado para ir a Pandora, donde algunas empresas están extrayendo un mineral extraño que podría resolver la crisis energética de la Tierra. Para contrarrestar la toxicidad de la atmósfera de Pandora, se ha creado el programa Avatar, gracias al cual los seres humanos mantienen sus conciencias unidas a un avatar: un cuerpo biológico controlado de forma remota que puede sobrevivir en el aire letal. Esos cuerpos han sido creados con ADN humano mezclado con ADN de los nativos de Pandora, los Na'vi. Convertido en avatar, Jake puede caminar otra vez. Su misión consiste en infiltrarse entre los Na'vi, que se han convertido en el mayor obstáculo para la extracción del mineral. Pero cuando Neytiri, una bella Na'vi, salva la vida de Jake, todo cambia: Jake, tras superar ciertas pruebas, es admitido en su clan.",
      "popularity": 41.077,
      "poster_path": "/pgjs6VwqKa9COml98tqMGjIkoJG.jpg",
      "release_date": "2009-12-17",
      "title": "Avatar",
      "video": false,
      "vote_average": 7.4,
      "vote_count": 21476,
    },
    Object {
      "adult": false,
      "backdrop_path": "/xTLl2qNYGIEKxqkqKdL5VuR3Er2.jpg",
      "genre_ids": Array [
        12,
        10751,
        16,
        28,
        35,
      ],
      "id": 177572,
      "original_language": "en",
      "original_title": "Big Hero 6",
      "overview": "En la metrópolis de San Fransokyo (cruce de San Francisco y Tokio), vive Hiro Hamada, quien aprende a sacar provecho de su capacidad gracias a su brillante hermano Tadashi y sus también brillantes amigos: la buscadora de adrenalina GoGo Tamago, el meticuloso de la limpieza Wasabi-No-Ginger, la genia de la química Honey Lemon y el fanático de los cómics Fred. Cuando tras un devastador giro de los acontecimientos, se ven envueltos en una peligrosa conspiración que tiene lugar en las calles de San Fransokyo, Hiro recurre a su amigo más íntimo: un robot llamado Baymax, y transforma al grupo en una banda de héroes de última tecnología decididos a resolver el misterio. Adaptación animada del cómic de Marvel.",
      "popularity": 39.884,
      "poster_path": "/dttSIaRB0D06AVL2Mz7ilnxdI8q.jpg",
      "release_date": "2014-12-24",
      "title": "Big Hero 6",
      "video": false,
      "vote_average": 7.8,
      "vote_count": 11738,
    },
    Object {
      "adult": false,
      "backdrop_path": "/p2fRZzxla6NoRbIH2KOZq0gHb5S.jpg",
      "genre_ids": Array [
        12,
        16,
        35,
        10751,
      ],
      "id": 269149,
      "original_language": "en",
      "original_title": "Zootopia",
      "overview": "La moderna metrópoli mamífera de Zootrópolis es una ciudad absolutamente única. Está compuesta de barrios con diferentes hábitats como la lujosa Plaza Sahara, el Distrito Selva Tropical y el gélido Distrito Tundra. Es un crisol donde los animales de cada entorno conviven, un lugar donde no importa lo que seas. De hecho puedes ser cualquier cosa, desde un elefante enorme hasta la musaraña más diminuta. Pero cuando llega la optimista agente Judy Hopps, descubre que ser la primera conejita de un cuerpo policial compuesto de animales duros y enormes no es nada fácil. Pero está decidida a demostrar su valía y se mete de cabeza en un caso, a pesar de que eso significa trabajar con Nick Wilde, un zorro parlanchín y estafador, para resolver el misterio.",
      "popularity": 39.307,
      "poster_path": "/i3WnZpJkQYXXbSGplEGmyoyeskM.jpg",
      "release_date": "2016-02-18",
      "title": "Zootrópolis",
      "video": false,
      "vote_average": 7.7,
      "vote_count": 11767,
    },
  ],
  "total_pages": 49,
  "total_results": 974,
}
`;

exports[`snapshots v4 should return a valid response from movie popular 1`] = `
Object {
  "page": 1,
  "results": Array [
    Object {
      "adult": false,
      "backdrop_path": "/8iVyhmjzUbvAGppkdCZPiyEHSoF.jpg",
      "genre_ids": Array [
        18,
      ],
      "id": 550,
      "original_language": "en",
      "original_title": "Fight Club",
      "overview": "A ticking-time-bomb insomniac and a slippery soap salesman channel primal male aggression into a shocking new form of therapy. Their concept catches on, with underground \\"fight clubs\\" forming in every town, until an eccentric gets in the way and ignites an out-of-control spiral toward oblivion.",
      "popularity": 24.594,
      "poster_path": "/bptfVGEQuv6vDTIMVCHjJ9Dz8PX.jpg",
      "release_date": "1999-10-15",
      "title": "Fight Club",
      "video": false,
      "vote_average": 8.4,
      "vote_count": 19383,
    },
  ],
  "total_pages": 1,
  "total_results": 1,
}
`;
